packages/cli/src/commands/rollback/index.ts

```diff
@@ -35,10 +35,14 @@ export default async (client: Client): Promise<number> => {
     return 1;
   }
 
+  const actionOrDeployId = parsedArgs.args[1] || 'status';
+
   telemetry.trackCliOptionTimeout(parsedArgs.flags['--timeout']);
   telemetry.trackCliFlagYes(parsedArgs.flags['--yes']);
 
   if (parsedArgs.flags['--help']) {
+    const subcommand = actionOrDeployId === 'status' ? 'status' : undefined;
```

Comment:
rollback technically has a subcommand (`status`) and a default, but is not modeled that way, so it's help tracking implementation deviates from the patterns elsewhere.